src/components/Label/Label.tsx

```diff
@@ -53,12 +57,30 @@ const Label: FC<LabelProps> & {
           [styles.withAnimation]: !isAnimationDisabled,
           [styles.withLeg]: isLegIncluded
         },
-        labelClassName
+        labelClassName,
+        { [styles.clickable]: isClickable }
       ),
-    [kind, color, isAnimationDisabled, isLegIncluded, labelClassName]
+    [kind, color, isAnimationDisabled, isLegIncluded, labelClassName, isClickable]
   );
+  const labelRef = useRef<HTMLSpanElement>(null);
+  const clickableProps = useClickableProps(
+    {
+      onClick,
```

Comment:
Let's consider adding `preventDefault` to the `useClickableProps` `onClick` callback - it would remove the jump when the element is pressed by Space button. As far as I know, it's the best practice